Swift/Tests/CBLTestCase.swift

```diff
@@ -60,6 +60,8 @@ class CBLTestCase: XCTestCase {
     override func setUp() {
         super.setUp()
         
+        Database.log.console.level = .info
```

Comment:
planned to remove?

Reply:
I think about keeping it as now we didn't log anything when running Swift Tests.

Reply:
logs might be noisy? but i guess it should be okay.